lib/danger/helpers/comments_helper.rb

```diff
@@ -104,7 +104,7 @@ def generate_comment(warnings: [], errors: [], messages: [], markdowns: [], prev
         )
       end
 
-      def generate_inline_comment_body(emoji, message, danger_id: "danger", resolved: false, template: "github")
+      def generate_inline_comment_body(emoji, message, danger_id: "danger", resolved: [], template: "github")
```

Comment:
I changed this because it seemed like resolved was never tested for falsiness - and always assumed to be an array in the templates.